### DISCUSSION TITLE
SKIOVOX 118 Question

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#904] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
So in step 2A.4 when it says to make a new window it doesn't show up.  So how? Also the 120 version way doesn't work either?Any ways to fix this? (Btw if you are asking how I'm in 118 my friend did smth with a USB so he could downgrade)

**Comment by S-PScripts** [comment]
necropost to answer
